doc/source/operating.rst

```diff
@@ -227,62 +227,63 @@ metricd daemon on any number of servers.
+each sack is assigned to one or more `gnocchi-metricd` daemons for
+processing.
 
-How many sacks do we need to create
------------------------------------
-
-This number of sacks enabled should be set based on the number of active
-|metrics| the system will capture. Additionally, the number of sacks, should
-be higher than the total number of active `gnocchi-metricd` workers.
+The number of sacks should be set based on the number of active
+|metrics| the system will capture. Additionally, the number of sacks
+should be higher than the total number of active `gnocchi-metricd`
+workers.
 
 In general, use the following equation to determine the appropriate `sacks`
 value to set::
 
    sacks value = number of **active** metrics / 300
 
-If the estimated number of |metrics| is the absolute maximum, divide the value
-by 500 instead. If the estimated number of active |metrics| is conservative and
-expected to grow, divide the value by 100 instead to accommodate growth.
+If the estimated number of |metrics| is the absolute maximum, divide
+the value by 500 instead. If the estimated number of active |metrics|
+is conservative and expected to grow, divide the value by 100 instead
+to accommodate growth.
 
 How do we change sack size
 --------------------------
 
-In the event your system grows to capture signficantly more |metrics| than
-originally anticipated, the number of sacks can be changed to maintain good
-distribution. To avoid any loss of data when modifying `sacks` option. The
-option should be changed in the following order::
+In the event your system grows to capture significantly more |metrics|
+than originally anticipated, the number of sacks can be changed to
+maintain good distribution. To avoid any loss of data when modifying
+the number of `sacks`, the value should be changed in the following
+order:
 
-  1. Stop all input services (api, statsd)
+1. Stop all input services (api, statsd).
 
-  2. Stop all metricd services once backlog is cleared
+2. Stop all metricd services once backlog is cleared.
 
-  3. Run gnocchi-change-sack-size <number of sacks> to set new sack size. Note
-     that sack value can only be changed if the backlog is empty.
+3. Run ``gnocchi-change-sack-size <number of sacks>`` to set new sack
+   size. Note that the sack value can only be changed if the backlog
+   is empty.
 
-  4. Restart all gnocchi services (api, statsd, metricd) with new configuration
+4. Restart all gnocchi services (api, statsd, metricd) with the new
+   configuration.
 
-Alternatively, to minimise API downtime::
+Alternatively, to minimize API downtime:
 
-  1. Run gnocchi-upgrade but use a new incoming storage target such as a new
-     ceph pool, file path, etc... Additionally, set |aggregate| storage to a
-     new target as well.
+1. Run gnocchi-upgrade but use a new incoming storage target such as a new
+   ceph pool, file path, etc. Additionally, set |aggregate| storage to a
+   new target as well.
```

Comment:
Do we want to keep that "additionally" ?

Reply:
Based on the irc discussion, it sounds like targeting new aggregate storage in this step may reduce the run-time and concomitant i/o of running the command against active aggregate storage. If that's not actually the case then sure, we should probably remove it.